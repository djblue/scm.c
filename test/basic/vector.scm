(define vec #(1 2 3))
(display vec)
(display (vector? 1))
(display (vector? "hello"))
(display (vector? '(1 2 3)))
(display (vector? vec))
(display (vector-length vec))
(display (vector-ref vec 0))
(display (vector-ref vec 1))
(display (vector-ref vec 2))
(vector-set! vec 0 4)
(vector-set! vec 1 5)
(vector-set! vec 2 6)
(display vec)
(display (vector-length vec))
(display (vector-ref vec 0))
(display (vector-ref vec 1))
(display (vector-ref vec 2))
