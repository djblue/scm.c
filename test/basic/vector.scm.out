#(1 2 3)
#f
#f
#f
#t
3
1
2
3
#(4 5 6)
3
4
5
6
